lib/request/validator.dart

```diff
@@ -1,19 +1,59 @@
+import 'dart:convert';
+import 'dart:math';
+
 import 'package:shelf/shelf.dart';
 
 Future<dynamic> checkRequest(Request request) async {
-  if (request.method != 'GET') {
```

Comment:
Ông add lại cái check mà method ko phải GET hay POST thì return 405 Method Not Allowed đc ko ông?

Reply:
để mấy cái check này ở trên top cho nó dễ đọc code á ông, h tôi lướt xuống dưới ms thấy else return 405 :>